--- logs/package.json.098	2026-08-28 12:59:50.416699824 +0000
+++ logs/package.json.099a1	2026-08-28 13:00:37.421973391 +0000
@@ -7,27 +7,26 @@
   },
   "dependencies": {
     "@radix-ui/themes": "3.3.0",
-    "@react-router/node": "7.18.2",
+    "@react-router/node": "8.3.0",
     "isbot": "5.2.1",
     "lucide-react": "1.26.0",
     "react": "19.2.8",
     "react-dom": "19.2.8",
     "react-error-boundary": "6.1.2",
     "react-helmet": "6.1.0",
-    "react-router": "7.18.2",
-    "react-router-dom": "7.18.2",
+    "react-router": "8.3.0",
     "socket.io-client": "4.8.3",
     "sonner": "2.0.7",
     "universal-cookie": "8.1.2"
   },
   "devDependencies": {
     "@emotion/react": "11.14.0",
-    "@react-router/dev": "7.18.2",
-    "@react-router/fs-routes": "7.18.2",
+    "@react-router/dev": "8.3.0",
+    "@react-router/fs-routes": "8.3.0",
     "autoprefixer": "10.5.4",
     "postcss": "8.5.23",
     "postcss-import": "16.1.1",
-    "vite": "8.0.16"
+    "vite": "8.2.0"
   },
   "overrides": {
     "postcss": "8.5.23"
